manifests/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: kepler-model-server-pv
+  labels:
+    type: local
+    app.kubernetes.io/component: kepler-model-server-pv
+    app.kubernetes.io/name: kepler-model-server-pv
+spec:
+  storageClassName: default
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/data"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kepler-model-server-pvc
+  namespace: monitoring
+spec:
+  storageClassName: default
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kepler-model-server
+  namespace: monitoring
+  labels:
+    app.kubernetes.io/component: model-server
+    app.kubernetes.io/name: kepler-model-server
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: model-server
+      app.kubernetes.io/name: kepler-model-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: model-server
+        app.kubernetes.io/name: kepler-model-server
+    spec:
+      volumes:
+      - name: models
+        persistentVolumeClaim:
+          claimName: kepler-model-server-pvc
+      containers:
+      - name: kepler-model-server
+        image: quay.io/sustainable_computing_io/kepler_model_server_base:latest
+        ports:
+        - containerPort: 8100
+        volumeMounts:
+          - name: models
+            mountPath: /models
```

Comment:
the mountPath needs to match the work dir so the models are saved in the PV. Or change the path defined [here ](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/server/kepler_model_trainer.py#L11) as absolute path  and match the mountPath